pkg/apis/kops/cluster.go

```diff
@@ -324,8 +325,10 @@ type ClusterSubnetSpec struct {
 	Name string `json:"name,omitempty"`
 	// CIDR is the network cidr of the subnet
 	CIDR string `json:"cidr,omitempty"`
-	// Zone is the zone the subnet resides
+	// Zone is the zone the subnet is in, set for subnets that are zonally scoped
```

Comment:
Can we just have one value to use across all providers?

Reply:
Not sure I understand